src/ScottPlot5/ScottPlot5/FontService.cs

```diff
@@ -34,9 +35,9 @@ private static SKTypeface GetDefaultSansFont()
         return SKTypeface.Default;
     }
 
-    private static SKTypeface GetDefaultMonospaceFont()
+    public static SKTypeface GetDefaultMonospaceFont()
     {
-        HashSet<string> installedFonts = new(SKFontManager.Default.FontFamilies);
+        var installedFonts = GetInstalledFonts();
 
         string[] preferredFonts = { "Roboto Mono", "Consolas", "DejaVu Sans Mono", "Courier" };
```

Comment:
why not use system by default？

Reply:
Great question! Most Windows systems use Segoe UI by default. I find the anti-alising looks bad. Look at the "V" in the word "Vertical"

![image](https://github.com/ScottPlot/ScottPlot/assets/4165489/2cd05979-1f0f-4b90-9c17-3902322324f3)

Compare to Open Sans which looks fantastic, and I prefer as a default

![image](https://github.com/ScottPlot/ScottPlot/assets/4165489/d212b17b-e81c-4ad6-842a-670556135a3b)